screens/AnnouncementScreen.js

```diff
@@ -41,6 +43,12 @@ class AnnouncementScreen extends React.Component {
     };
   }
 
+  async componentDidMount() {
+    Font.loadAsync({
+      Poppins: require('../assets/fonts/Poppins-Regular.ttf')
+    });
+  }
+
   render() {
     return (
       <View>
```

Comment:
Nitpick: We don't need the 'Events' tab at all anymore; check the 'Week 15' in Figma while you are styling please!